index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">Sajeel and Abel</h1>
+      <div id="img-div">
+          <img id="image" src="https://image.shutterstock.com/image-photo/islamabad-pakistan-april-25-2019-260nw-1407461093.jpg" >
+          <figcaption id="img-caption">Landscape</figcaption>
+          <p id="tribute-info">This landscape is very beautiful. It is found in Islamabad, Pakistan.</p>
+          <a href="https://image.shutterstock.com/image-photo/islamabad-pakistan-april-25-2019-260nw-1407461093.jpg" id="tribute-link" target="_blank">Image Link</a>
```

Comment:
In my opinion, it's better to use semantic html5 tags instead of div.

Reply:
> In my opinion, it's better to use semantic html5 tags instead of div.

- [ ] Always try to be clear and specific when requesting a change. In this case, you should state which semantic HTML tags should be used for example `<section>`, `<header>` or `<nav>` and where exactly in the code. This would enable the other person to get a better understanding of the issue. Also remember to provide a reason as to `why ?` you are requesting the change
- [ ]  Would you please provide at least two comment reviews